src/components/ucp-tabs/extension-manager/extension-elements/extension-element.tsx

```diff
@@ -319,7 +317,9 @@ export function CustomisationsExtensionElement() {
       {trashButton}
       <div className="extension-name-box">
         {/* eslint-disable-next-line jsx-a11y/click-events-have-key-events, jsx-a11y/no-static-element-interactions */}
-        <span className="extension-name-box__name">{t('Customisations')}</span>
+        <span className="extension-name-box__name">
+          <Message message="Customisations" />
```

Comment:
So this line is the cause of the error I shared the screenshot for. I tried to implement a localized message like so:

```tsx
          <Message key="extensions.customisations" />
```
and I added `extensions.customisations: "Customisations"` to en.yaml

However, the GUI element stays blank, no text! (Also after GUI restart)
![image](https://github.com/user-attachments/assets/3fa14352-a778-4f88-9458-04a485aca37a)

What is the proper way of doing it?

Reply:
Yea that is why I restarted the gui using `npm run dev`, but still it remains blank! I can try debug the InnerMessage and check for the key to not break on all uses.

Reply:
Yea, please debug, maybe you found a bug. A missing message should produce the key. An empty result should only appear if the localization is intentionally an empty string. It should be relatively easy to debug into the resolving function, although, you might need to reload to avoid memoization. All keys are just loaded as yaml and are put into an object.